src/MassTransit.RabbitMqTransport/IRoutingkeyFormatter.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace MassTransit.RabbitMqTransport
+{
+    public interface IRoutingkeyFormatter
+    {
+        string createRoutingkeyForType(Type messageType);
```

Comment:
`CreateRoutingKeyForType`. Start with a capital, routing and key are separate words.